projects/hashking/index.js

```diff
@@ -7,7 +7,11 @@ module.exports = {
       const validators = await api.call({ target: owner, abi: 'function beneficiarys() public view returns (address [] memory)'})
       const bals = await api.multiCall({  abi: 'uint256:totalStakingFil', calls: validators})
       bals.forEach(i => api.add(nullAddress, i))
-      return sumTokens2({ api, owners: [owner], tokens: [nullAddress]})
+      const wFILContract = "0xD9A724840a46370c01a50C1E511087ab3a07FB53"
+      const totalSupply = await api.call({ target: wFILContract, abi: 'uint256:totalSupply'})
```

Comment:
why are you counting total wFIL supply as tvl?

Reply:
wFIL would represent the amount in our liquidity pool.

Reply:
> wFIL would represent the amount in our liquidity pool.

isnt wFIL like WETH? why does that represent your liquidity pool, I am confused
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/99907941/d95e46e1-7a22-4000-996a-f5ffac75bdad)